ciri: does not compile on MacOs

```
ERROR: /Users/thibault/iota/entangled/ciri/BUILD:6:1: Linking of rule '//ciri:ciri' failed (Exit 1) cc_wrapper.sh failed: error executing command external/local_config_cc/cc_wrapper.sh -lc++ -fobjc-link-runtime -o bazel-out/darwin-opt/bin/ciri/ciri bazel-out/darwin-opt/bin/ciri/_objs/ciri/main.o bazel-out/darwin-opt/bin/ciri/libcore.a ... (remaining 110 argument(s) skipped)

Use --sandbox_debug to see verbose messages from the sandbox
Undefined symbols for architecture x86_64:
  "_json_logger_id", referenced from:
      _json_add_neighbors_serialize_request in libserializer_json.a(add_neighbors.o)
      _json_add_neighbors_serialize_response in libserializer_json.a(add_neighbors.o)
      _json_add_neighbors_deserialize_request in libserializer_json.a(add_neighbors.o)
      _json_add_neighbors_deserialize_response in libserializer_json.a(add_neighbors.o)
      _json_attach_to_tangle_serialize_request in libserializer_json.a(attach_to_tangle.o)
      _json_attach_to_tangle_serialize_response in libserializer_json.a(attach_to_tangle.o)
      _json_attach_to_tangle_deserialize_request in libserializer_json.a(attach_to_tangle.o)
      ...
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
```

--- a/cclient/serialization/BUILD
+++ b/cclient/serialization/BUILD
@@ -12,6 +12,7 @@ cc_library(
         "@cJSON",
         "@com_github_uthash//:uthash",
     ],
+    alwayslink = True,
 )
 
 cc_library(

